spec/models/king_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe King, type: :model do
+	let(:king) {create :king}
```

Comment:
Due to the associations between these models, creating an instance of `King` will also create a game, and that will in turn trigger the creation of 32 pieces via `populate_board!`, making the original king redundant. Instead, I would set up each test by creating a game and then selecting one of the kings associated with that game via `find_by`.